docs/source/sysadmin_guide/configuring_sawtooth/xo_tp_configuration.rst

```diff
@@ -15,8 +15,8 @@ file.
    By default, the config directory is ``/etc/sawtooth/``.
    See :doc:`path_configuration_file` for more information.
 
-An example configuration file is in the ``sawtooth-core`` repository at
-``/sawtooth-core/sdk/examples/xo_python/packaging/xo.toml.example``.
+An example configuration file is in the ``sawtooth-sdk-python`` repository at
+``https://github.com/hyperledger/sawtooth-sdk-python/blob/add-sdk/examples/xo_python/packaging/xo.toml.example``.
```

Comment:
`add-sdk` seems to be private branch. This URL becomes invalid for rest of the public.

Here's what I see as URL for the file https://github.com/hyperledger/sawtooth-sdk-python/blob/master/examples/xo_python/packaging/xo.toml.example